packages/itwinui-react/src/core/Breadcrumbs/BreadcrumbsItem.tsx

```diff
@@ -0,0 +1,125 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import cx from 'classnames';
+import { useTheme, type CommonProps } from '../utils/index.js';
+import '@itwin/itwinui-css/css/breadcrumbs.css';
+
+type BreadcrumbsAnchorItemProps =
+  | {
+      /**
+       * Type of breadcrumbs item.
+       * @default 'button'
+       */
+      type?: 'anchor';
+      /**
+       * Include an href for anchor type breadcrumbs.
+       * Use only when `type = 'anchor'`.
+       */
+      href?: string;
+    }
+  | {
+      /**
+       * Type of the breadcrumbs item.
+       * @default 'button'
+       */
+      type?: 'button' | 'text';
+    };
+
+type BreadcrumbsButtonItemProps =
+  | {
+      /**
+       * Type of the breadcrumbs item.
+       * @default 'button'
+       */
+      type?: 'button';
+      /**
+       * Include an onClick function for button type breadcrumbs.
+       * Use only when `type = 'button'`.
+       */
+      onClick?: () => void;
+    }
+  | {
+      /**
+       * Type of the breadcrumbs item.
+       * @default 'button'
+       */
+      type?: 'anchor' | 'text';
+    };
+
+export type BreadcrumbsItemProps = {
+  /**
+   * Type of the breadcrumbs item.
+   * @default 'button'
+   */
+  type?: 'anchor' | 'button' | 'text';
+  /**
+   * Include an href for anchor type breadcrumbs.
+   */
+  href?: string;
+  /**
+   * Include an onClick function for button type breadcrumbs.
+   */
+  onClick?: () => void;
```

Comment:
I think that I should be able to remove the three properties `type`, `href`, and `onClick` here because they're defined above (like in Tabs.tsx). I get an error below when defining the props though that it can't find those properties. Not sure what I'm doing wrong. I think the way that I'm currently doing it isn't good because it doesn't warn that you can't use href with buttons, etc.